docs/Docker.md

```diff
@@ -0,0 +1,36 @@
+# Docker
+
+## Developing in Docker
+
+If running for the first time, do:
+
+1. Delete local `node_modules` folder if it exists
+2. Run `docker-compose build`
+3. Run `docker-compose run groupincome npm install`
```

Comment:
I got a big diff in package-lock.json because of this:
```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```
did that also happen with your setup?

Reply:
Yes, I got those comments and yes also package-lock.json got updated, but that's because of issue #340, see the commentary there.